hack/windows/Clean-AntreaNetwork.ps1

```diff
@@ -74,10 +92,27 @@ function ResetOVSService() {
         ovsdb-tool create "$OVS_DB_PATH" "$OVS_DB_SCHEMA_PATH"
 
         start-service ovsdb-server
-        sc.exe create ovs-vswitchd binpath="$OVSInstallDir\usr\sbin\ovs-vswitchd.exe  --pidfile -vfile:info --log-file  --service" start= auto depend= "ovsdb-server"
-        sc.exe failure ovs-vswitchd reset= 0 actions= restart/0/restart/0/restart/0
-        start-service ovs-vswitchd
+        CreateStartOVSvSwitchd
+}
+
+function CreateStartOVSDB() {
+    $OVS_DB_SCHEMA_PATH = "$OVSInstallDir\usr\share\openvswitch\vswitch.ovsschema"
+    $OVS_DB_PATH = "$OVSInstallDir\etc\openvswitch\conf.db"
+    if ($(Test-Path $OVS_DB_SCHEMA_PATH) -and !$(Test-Path $OVS_DB_PATH)) {
+        Log "Creating ovsdb file"
+        ovsdb-tool create "$OVS_DB_PATH" "$OVS_DB_SCHEMA_PATH"
     }
+    sc.exe create ovsdb-server binPath= "$OVSInstallDir\usr\sbin\ovsdb-server.exe $OVSInstallDir\etc\openvswitch\conf.db  -vfile:info --remote=punix:db.sock  --remote=ptcp:6640  --log-file  --pidfile --service" start= auto
+    sc.exe failure ovsdb-server reset= 0 actions= restart/0/restart/0/restart/0
+    Start-Service ovsdb-server
+}
+
+function CreateStartOVSvSwitchd() {
+    sc.exe create ovs-vswitchd binpath="$OVSInstallDir\usr\sbin\ovs-vswitchd.exe  --pidfile -vfile:info --log-file  --service" start= auto depend= "ovsdb-server"
+    sc.exe failure ovs-vswitchd reset= 0 actions= restart/0/restart/0/restart/0
+    start-service ovs-vswitchd
+    $OVS_VERSION=$(Get-Item $OVSInstallDir\driver\OVSExt.sys).VersionInfo.ProductVersion
+    ovs-vsctl --no-wait set Open_vSwitch . ovs_version=$OVS_VERSION
```

Comment:
Why we need to set ovs_version here? Will restarting ovs service lose version info?

Reply:
The data is not lost if only restart ovs-vswitchd process/service. But if we re-create the OVS schema file ( called in line 92 when ovs-vswitchd failed ), this data is lost as we have deleted db.conf . This version is needed by in antrea-agent monitoring CR, and it may cause antrea-agent crash if not exists.